examples/js/ThreeLoader.js

```diff
@@ -0,0 +1,71 @@
+/* global itowns, Promise */
+var manager = new itowns.THREE.LoadingManager();
+
+/**
+ * Load a script asynchronously
+ * @param {String} uri URI of the script to load.
+ * @returns {Promise} A promise resolved when the script is loaded.
+ */
+function loadScriptAsync(uri) {
+    return new Promise(function p(resolve) {
+        var firstScriptTag = document.getElementsByTagName('script')[0];
+        var tag = document.createElement('script');
+        tag.src = uri;
+        tag.async = true;
+        tag.onload = function r() {
+            resolve();
+        };
+        firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);
+    });
+}
+
+/**
+ * Load script asynchronously from ThreeJs example loaders
+ *
+ * @param {String} format Format to be loader. Example: Collada will return ColladaLoader
+ * @returns {Promise} A promise with the ThreeJS Loader
+ */
+function getThreeJsLoader(format) {
+    var res;
+    var promise = new Promise(function p(resolve) {
+        res = resolve;
+    });
+    // eslint-disable-next-line no-undef
+    THREE = itowns.THREE;
+    loadScriptAsync('https://cdn.rawgit.com/mrdoob/three.js/r' + itowns.THREE.REVISION + '/examples/js/loaders/' + format + 'Loader.js')
```

Comment:
you could add cache to get loader if it is already loaded

Reply:
The developer can use ThreeLoader.getThreeJsLoader if he wants to re use the loader.
So I think a cache is not needed.